src/components/Form.js

```diff
@@ -5,15 +5,22 @@ import "../styles/components/form.scss";
 const Form = (props) => {
```

Comment:
you can use destructured props directly in the function `const Form = (data:{ title, paragraphs, textarea, link })`

Reply:
Type annotations are supported by TS, so they won't work here without additional libs.

Reply:
Handy.